modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -60,424 +61,56 @@ with links to every release package along with precompiled binaries.
+* Added support for Ruby 3.1.
 
-* https://issues.couchbase.com/browse/RCBC-311[RCBC-311]:
-  Fixed scope qualifer encoding for analtyics to work with latest decoding.
+* Dropped support for Ruby 2.5 and 2.6.
 
-* Dropped support of Ruby 2.5.
+Fixes:
+
+* https://issues.couchbase.com/browse/RCBC-358[RCBC-358]:
+  initialize search locations only if they returned by server
 
-* Many smaller fixes and improvements.
 
 [cols="9,5,19"]
 |===
 | Platform           | Ruby ABI | File
 | Checksums          |          | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0.sha256sum[couchbase-3.2.0.sha256sum]
 | Source Archive     |          | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0.gem[couchbase-3.2.0.gem]
+| Linux x86_64       | 3.1.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-linux-3.1.0.gem[couchbase-3.2.0-x86_64-linux-3.1.0.gem]
 | Linux x86_64       | 3.0.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-linux-3.0.0.gem[couchbase-3.2.0-x86_64-linux-3.0.0.gem]
 | Linux x86_64       | 2.7.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-linux-2.7.0.gem[couchbase-3.2.0-x86_64-linux-2.7.0.gem]
-| Linux x86_64       | 2.6.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-linux-2.6.0.gem[couchbase-3.2.0-x86_64-linux-2.6.0.gem]
-| Linux x86_64       | 2.5.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-linux-2.5.0.gem[couchbase-3.2.0-x86_64-linux-2.5.0.gem]
+| macOS 10.15 x84_64 | 3.1.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-19-3.1.0.gem[couchbase-3.2.0-x86_64-darwin-19-3.1.0.gem]
 | macOS 10.15 x84_64 | 3.0.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-19-3.0.0.gem[couchbase-3.2.0-x86_64-darwin-19-3.0.0.gem]
 | macOS 10.15 x84_64 | 2.7.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-19-2.7.0.gem[couchbase-3.2.0-x86_64-darwin-19-2.7.0.gem]
-| macOS 10.15 x84_64 | 2.6.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-19-2.6.0.gem[couchbase-3.2.0-x86_64-darwin-19-2.6.0.gem]
-| macOS 10.15 x84_64 | 2.5.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-19-2.5.0.gem[couchbase-3.2.0-x86_64-darwin-19-2.5.0.gem]
+| macOS 11 x84_64 | 3.1.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-20-3.1.0.gem[couchbase-3.2.0-x86_64-darwin-20-3.1.0.gem]
 | macOS 11 x84_64 | 3.0.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-20-3.0.0.gem[couchbase-3.2.0-x86_64-darwin-20-3.0.0.gem]
 | macOS 11 x84_64 | 2.7.0    | https://packages.couchbase.com/clients/ruby/sdk-3.2.0/couchbase-3.2.0-x86_64-darwin-20-2.7.0.gem[couchbase-3.2.0-x86_64-darwin-20-2.7.0.gem]
```

Comment:
the links are not available yet. I will leave comment here once I publish the artifacts to S3

Reply:
I think It's fine if the docs get published a bit sooner, they will eventually work once they are on S3, so I'm merging this 👍 .